src/menv/commands/internal/aider.py

```
@@ -60,104 +42,28 @@ def run(
-        completed = subprocess.run(full_command, check=False)
-    except FileNotFoundError as exc:
-        err_console.print(f"Error: failed to execute {full_command[0]!r}: {exc}")
-        raise typer.Exit(1)
-    raise typer.Exit(completed.returncode)
+    typer.echo(err_console_msg, err=True)
+    raise typer.Exit(1)
 
 
 @aider_app.command("set-model")
 def set_model(
     model: Optional[str] = typer.Argument(None, help="Ollama model name."),
```

Ruff (0.15.2)

[warning] Unused function argument: `model`

(ARG001)